app-misc/pwsafe/pwsafe-0.2.0-r5.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A Password Safe compatible command-line password manager"
+HOMEPAGE="https://github.com/nsd20463/pwsafe"
+SRC_URI="http://web.archive.org/web/20171006105548if_/http://nsd.dyndns.org/${PN}/releases/${P}.tar.gz"
```

Comment:
`https`

Since it is mirrored, I wouldn't bother updating SRC_URI. But have you checked that the manifests match?

There's also no need to use `${PN}` in SRC_URI since it's not a changeable variable.

Reply:
> `https`

Changed.

> Since it is mirrored, I wouldn't bother updating SRC_URI.

Does that mean you want me to change it back? I can do that, but @thesamesam seemed to prefer that SRC_URI be fetchable.

> But have you checked that the manifests match?

They do.

> There's also no need to use `${PN}` in SRC_URI since it's not a changeable variable.

Sorry, I saw hundreds of other packages doing that so I thought it was common practice. Changed.

> Sorry but looking at all these patches, it looks more complicated than a single "revbump".

I didn’t add any new patches. Those new files are patches that already applied to -r4, just with their headers changed from p0 format to p1 format, as @thesamesam requested so that I could use the `PATCHES` array instead of calls to `eapply`. `git show -C -C` proves that only the filename headers changed when I copied those patch files, none of the content.

Reply:
> 
> > Since it is mirrored, I wouldn't bother updating SRC_URI.
> 
> Does that mean you want me to change it back? I can do that, but @thesamesam seemed to prefer that SRC_URI be fetchable.
> 
> > But have you checked that the manifests match?
> 
> They do.

Both are fine for me especially if the source hash matches.

> 
> > There's also no need to use `${PN}` in SRC_URI since it's not a changeable variable.
> 
> Sorry, I saw hundreds of other packages doing that so I thought it was common practice. Changed.

Don't worry, it's not a strict rule really. Just a "common sense" one, 
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0103
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D

> 
> > Sorry but looking at all these patches, it looks more complicated than a single "revbump".
> 
> I didn’t add any new patches. Those new files are patches that already applied to -r4, just with their headers changed from p0 format to p1 format, as @thesamesam requested so that I could use the `PATCHES` array instead of calls to `eapply`. `git show -C -C` proves that only the filename headers changed when I copied those patch files, none of the content.

I'm sorry but that makes little sense to me, maybe the idea was that the previous files could be immediately dropped, but it's not possible because the -r4 is stabilized. We don't want to duplicate patches, suddently you'd have a set of 2x5 identical files in the tree.

But worry not, I will merge this today cherry-picking your file from https://github.com/gentoo/gentoo/commit/b961b1b6d73672afa7ff714873de778a0800eefe and adding whatever other fixes you did. So no action required from you anymore, just patience. Thanks!